docs/Software_Developer_Guide/index.html

```diff
@@ -133,6 +133,7 @@ <h2 id="setting-up-jenkinsfile-inside-github-project-repository">Setting up <cod
 <h2 id="setting-credentials-for-webhook">Setting credentials for webhook:</h2>
 <p>Jenkins supports webhooks which can trigger the job from external sources such as GitHub. They work in a way such that, if a specified branch is committed or if a pull request is created, the specified job build starts running depending upon the trigger event which is set in build's configuration in jenkins.  </p>
 <p>This process requires GitHub credentials of owner of repository on which the webhook is to be set. These credentials can be safely added to jenkins without anyone (even administrator) seeing the passwords as follows.
+- Contact us for an account (create an issue on github).
```

Comment:
Can you put a format for the details required, like we discussed, name, resources required, github repo, which organziation you belong to, and I think it might be better now that I think about it to contact us by using the github email or create and issue and we can get back to them.

Reply:
I am adding it. It seems that contact us details was not included in the mkdocs gh-pages. Let me edit this, then I will create a new pull request for newly created mkdocs. Also there seems to be an issue with the template, the bullets were not shown correctly.

Reply:
I have added the details needed for "Contact us".